src/llm_tracekit/coralogix.py

```diff
@@ -48,18 +84,19 @@ def setup_export_to_coralogix(
         processors: Optional list of SpanProcessor instances to add to the tracer provider before the exporter processor.
     """
 
-    # Read environment variables as defaults if needed
-    if coralogix_token is None:
-        coralogix_token = os.environ["CX_TOKEN"]
-    if coralogix_endpoint is None:
-        coralogix_endpoint = os.environ["CX_ENDPOINT"]
-    if application_name is None:
-        application_name = os.environ["CX_APPLICATION_NAME"]
-    if subsystem_name is None:
-        subsystem_name = os.environ["CX_SUBSYSTEM_NAME"]
     if capture_content:
         enable_capture_content()
 
+    exporter_config = generate_exporter_config(
+        coralogix_token=coralogix_token,
+        coralogix_endpoint=coralogix_endpoint,
+        application_name=application_name,
+        subsystem_name=subsystem_name
+    )
+
+    if not exporter_config:
+        return
```

Comment:
this changes the behavior we had if parameters are not passed, let's keep it the same as it was (so return an ExporterConfig, potentially with null values inside)